lib/y2_logs_helper.pm

```diff
@@ -101,7 +101,6 @@ sub accept_license {
 }
 
 sub verify_license_translations {
-    return if (is_sle && get_var("BETA") || check_var('VIDEOMODE', 'text'));
```

Comment:
Logic here is that we might not have license properly translated till RC phase and such bugs are rejected, so I would keep this condition.

Reply:
bc62f1c58 reverted the `is_sle && get_var("BETA") `